imkar/scattering/scattering.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import pyfar as pf
+from imkar import utils
+
+
+def freefield(sample_pressure, reference_pressure, microphone_weights):
```

Comment:
It might be stupid me and the long while since I had to deal with computing scattering coefficients, but I'm confused with the following:

If I pass 10-channel, 3-frequencies data the function returns single-channel, 3-frequencies data, but the docstring mentions that the scattering is computed for each incidence. Can you make more clear what the angles _S and _R are and how they are encoded in the input and output data?

Reply:
Should we use `None` as a default that uses equal weights?

Reply:
good point, it does not need to be for each incident angle, because it depends on the input data, what ever is in the dimentions before the last one, gets retured, so I delete the reference for each incident angle.

if we use None, we need the data points instead, and it will get more complex,

Reply:
I ment `None` might be a good default to compute equal weights inside the function. But that might not make sense if equal weights are very uncommon for these measurements.